semeio/jobs/scripts/ots.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+from semeio.jobs.ots.ots import ots_run
+import argparse
+from semeio import valid_file
+
+description = (
+    "Overburden timeshift (OTS) generates evolution of reservoir surfaces"
+    "based on eclipse models and seismic velocity volume."
+    "Input yml needs to contains vintages section, where at"
+    " least one of the four categories"
+    "for the surface computation needs to be set."
+)
+
+category = "modeling.surface"
+
+
+def _get_args_parser():
```

Comment:
There's a good deal of information in the configsuite schema. It is possible to create a RST output of the content with `configsuite.create_rst` or something similar. Could you add an argument to the job that will output this information? Or add it as part of the description? Part of the description if probably nice, as this will be included on the docs pages.

Reply:
I would say that having a specific parameter for creating the rst file sounds like a very good idea!

Reply:
Currently I made it only that `configsuite.docs.generate(schema)` appends the string to the description. Do you think it suffice?

Reply:
In the end I had decided for an extra parameter `--params` just showing the RST output from configsuite.

Reply:
This should also be included in the description, as the top level `description` will be compiled as documentation for `ert` and that way it will be much more accessible for the user.